src/Courses/CourseDetailsTabs/OverrideGradeModal.tsx

```diff
@@ -35,6 +36,14 @@ export const OverrideGradeModal: React.FC<OverrideGradeModalProps> = ({
     const [loading, setLoading] = useState<boolean>(false);
     const [newScorePercentInput, setNewScorePercentInput] = useState<string>(displayCurrentScore.current ?? '');
 
+    useEffect(() => {
+        if (inputRef.current) {
+            inputRef.current.select();
+        }
+    // inputRef.current is set to a new value every time the dialog comes up
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [inputRef.current]);
```

Comment:
What's the exhaustive dep warning here? Is it to not watch inputRef.current?

Reply:
Yeah it says ref changes don't cause rerenders, but if the ref does change I want it to execute the code

This component gets mounted an unmounted